homeassistant/components/zha/core/endpoint.py

```diff
@@ -120,6 +118,15 @@ def add_all_cluster_handlers(self) -> None:
                 cluster_id, ClusterHandler
             )
 
+            if hasattr(cluster.endpoint.device, "quirk_id"):
```

Comment:
When https://github.com/home-assistant/core/pull/102482 gets merged, it might be better to use `endpoint.device.quirk_id` here.
It's `None` or a `str` with the quirk ID.

(There's a difference between the ZHA endpoint and zigpy endpoint, as well as the ZHA device and the zigpy device.)

Reply:
Thanks, I changed the code a little to work nicely with that assumption.